doltpy_integrations/metaflow/examples/hospital_bounty/blog_post.md

```diff
@@ -0,0 +1,170 @@
+            variance_by_procedure = median_price_by_state.groupby("code").var()
+            dolt.write(variance_by_procedure, "variance_by_procedure")
+
+    @step
+    def end(self):
+        pass
+```
+
+Let's access that result via the Meatflow API to make clear the mechanism of the integration:
+```
+from metaflow import Flow
+from doltpy_integrations.metaflow import 
+dolt = DoltDT(run=self, audit=Flow("HospitalPriceVariance").latest_successful_run)
+df = dolt.read("variance_by_procedure")
+# TODO show the dataset here
+```
+
+The `DoltDT` class knows how to use a Metaflow `Run` instance to resolve to a commit, and return the exact tables that were read or written. Let's move on to examine what this actually enables Metaflow users to do in practice.
+
+### Lineage
+Now suppose that we use the same method to access the medians we used to compute procedure variances across states:
+
+```
+from metaflow import Flow
+from doltpy_integrations.metaflow import 
+dolt = DoltDT(run=self, audit=Flow("HospitalPriceVariance").latest_successful_run)
+df = dolt.read("state_procedure_medians")
+```
+
+We just pulled in the intermediate dataset that was used to compute the variances directly from the Metaflow client. Now
+
+### Reproducibility
+Suppose now that we make a series of changes to our Metaflow job to correct for how we are computing the variances. The upstream dataset has been changing, but we want to verify that our code changes produce the desired results by fixing the input data:
+
+```
+> poetry run python3 hospital_price_variance.py run \ 
+--historical-run-path HospitalPriceVariance/XX
+--hospital-price-analysis-db path/to/hospital-price-analysis
+```
+
+We have now executed our updated code on an exact copy of the data our original analysis ran against. We can immediately examine the results and verify them:
+
+```
+from metaflow import Flow
+from doltpy_integrations.metaflow import 
+dolt = DoltDT(run=self, audit=Flow("HospitalPriceVariance").latest_successful_run)
+df = dolt.read("variance_by_procedure")
+# TODO we can show a diff here
+```
+
+### Back-testing
+Our input dataset is coming from Dolt, the hospital price transparency dataset. Suppose that we are training a model using this dataset. We need a training and testing dataset. We can do that by selecting a subset of commits on our upstream database, and use that selection to train and test our model:
+```
+# TODO sample commit graph and execute test/train
+```
+
+## Conclusion
+In this post we saw how to use Dolt from Metaflow. Dolt is a version controlled SQL database, and Metaflow is a framework or defining data engineering and data science jobs. The Metaflow + Dolt integration allows Metaflow users to use Dolt directly from the Metaflow API without having to learn new concepts. At the same time users that know Dolt, or SQL and Git, can use Dolt directly to manage and interpret the results of their Metaflow jobs that are written to their Dolt databases.
```

Comment:
I think you have to mention Lineage, Reproducibility, and Back Testing here. That's really what Metaflow users get.